build-chroot/debootstrap-1.0.106.deb.sha256

```diff
@@ -0,0 +1 @@
+14da318a90c66b9ef4be97b950bffd418033598c44f12f472d3501edec103f40  debootstrap-1.0.106.deb
```

Comment:
Can we compute this hash in `debian-checksum.sh`?

Reply:
There's no point in doing so; we can't update this until the bug gets fixed, and at that point, we won't need the pin anymore.